src/Reflection/Adapter/ReflectionClass.php

```diff
@@ -411,19 +411,20 @@ public function getParentClass()
     }
 
     /**
-     * {@inheritDoc}
+     * @psalm-suppress MethodSignatureMismatch
```

Comment:
Is this suppression due to psalm's stubs being out of sync?

Reply:
`MethodSignatureMismatch - src/Reflection/Adapter/ReflectionClass.php:417:61 - Argument 1 of Roave\BetterReflection\Reflection\Adapter\ReflectionClass::isSubclassOf has wrong type 'ReflectionClass|string', expecting 'ReflectionClass|class-string' as defined by ReflectionClass::isSubclassOf (see https://psalm.dev/042)`

I don't know how to solve it. Is there a way? `@param` does not work...